rtl/cTCLS_unit/cTCLS_unit.hjson

```diff
@@ -0,0 +1,59 @@
+{
+  name: "cTCLS_manager",
+  clock_primary: "clk_i",
+  reset_primary: "rst_ni",
+  bus_device: "reg",
+  bus_host: "",
+
+  regwidth: "32",
+  registers: [
+    { name: "MODE",
+      desc: "Redundancy Mode configuration",
+      swaccess: "wo",
+      hwaccess: "hro",
+      fields: [
+        { bits: "0",
+          name: "MODE",
+          desc: "TMR mode enable"
+        },
+        { bits: "8",
```

Comment:
This seems an extremely low-level optimization. In the end, the register read/writes will be done through a library or hardware abstraction layer that will most likely not know about this. Also, it's not expected that this will change very often. So I think it's not worth the trouble.